### ISSUE TITLE
v2 release

### ISSUE THREAD

**Issue by ming517** [yuche/vue-strap#435]
Hi,

May I know when is the release for v2 that compatible with Vue 2?

Or anyway I can install it via npm?

Thanks and so much appreciate the hard works for this plugin :)

**Comment by upupzealot** [comment]
+1

**Comment by tommyzqfeng** [comment]
+1

**Comment by s-mirrorman** [comment]
+1

**Comment by senid231** [comment]
+1
I'm interested in that too

**Comment by wffranco** [comment]
@yuche was inactive for a long time so the last version is in my branch.

Add this to your packaje.json if you want to use the last version:
```javascript
 "vue-strap": "github:wffranco/vue-strap.git"
```

**Comment by OmgImAlexis** [comment]
For those using yarn this works.
```
"vue-strap": "https://github.com/wffranco/vue-strap.git"
```

**Comment by gholol** [comment]
@OmgImAlexis Incompatible with vue 2.1.x

**Comment by OmgImAlexis** [comment]
Yeah I noticed that after spending a few hours trying to workout why I couldn't get even a single page to build using browserify. 😞

**Comment by 445141126** [comment]
@gholol vue 2.1.5 change render function name, so the dist js file not working.
just use ` import { navbar, dropdown } from 'vue-strap/src'`.

**Comment by wontwon** [comment]
+1

**Comment by wffranco** [comment]
Fixed for vue 2.1.x in my fork... but now is not compatible with vue 2.0.x

**Comment by gholol** [comment]
@wffranco Maybe make a new release for it? v 1.10 for 2.1.x onwards

**Comment by gholol** [comment]
@wffranco Can you confirm it works on ?

Tried your master branch and  input with aside components and they show "undefined" text. I also don't get any errors

http://i.imgur.com/lL2gSkj.png

**Comment by ndarilek** [comment]
So am I correct in assuming that, if I use this component directly from Git, it will work with Vue 2.1.x and Bootstrap 3.3.7? Or is this version in master still 1.x-only?

Thanks.

**Comment by jlkm2010** [comment]
Thanks for your time.
